### PR TITLE
Notification Center: avoid localStorage actions on SSR

### PR DESCRIPTION
**Pull request by 13colours**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Bug fix

- **What is the current behavior?** (You can also link to an open issue here)
`localStorage` is `undefined` on SSR
Ref: https://github.com/novuhq/novu/issues/560

- **What is the new behavior (if this is a feature change)?**
`localStorage` methods run only on browser environment

- **Other information**:

### CODE REVIEW

**Comment by 13colours** [comment]
I was unable to find any tests for notification-center package, so I have marked the PR as draft for now. Would appreciate some help to test this change. Thanks.

**Comment by davidsoderberg** [comment]
To me this looks great 🎉  I think @djabarovgeorge may be able to help you out with tests :)

**Comment by aaron5670** [comment]
Loooks great! 🙏

**Comment by djabarovgeorge** [comment]
Hey @davidsoderberg, we could test the notification-center through the app/widget cypress test because it contains the @novu/notification-center

**Comment by 13colours** [comment]
> Hey @davidsoderberg @13colours , we could test the notification-center through the app/widget cypress test because it contains the @novu/notification-center

Thanks @djabarovgeorge 
Think I need some help here. So, I ran the following 3 commands in separate tabs to run the cypress test in `apps/widget`. However, out of 4 specs, all failed except `initialization` spec. The tests were run in the `main` branch.
```
npm run start:test
npm run start:api
npm run cypress:open (had to change the script to "cypress:open": "cypress open", as it was throwing port conflict
```

Am I missing anything here?
Thanks!

**Comment by djabarovgeorge** [comment]
Hey @13colours 
Thanks for approaching I will do my best in order to help! :)

First of all, you are right regarding 'throwing port conflict' it was probably caused because both 'npm run start:test' and 'npm cypress:open' execute 'start:shell:test' that opens HTTP server on port.

The errors you get will help in order to understand better the issue you experience. my guess is there are missing running services. 

I will try to help you one this comment, please try to run the following command:

- in apps/api/package.json     run : start:test
- in apps/ws/package.json.    run: start:test
- in apps/widget/package.json    run: start:test

And then run the cypress (run or open). 

Thank you and please let us know if you need any further assistence

File: apps/widget/README.md [hunk 1]
```diff
@@ -1,46 +1,11 @@
...
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
+Once all the above scripts are running, run `npm run cypress:open` from apps/widget to view the Cypress Test Runner.
```
Comment (scopsy) [inline apps/widget/README.md:11]: This is such an amazing update! Thank you!

**Comment by scopsy** [comment]
This looks great! @djabarovgeorge if you could give it another look and maybe some sample testing and if everything is ok we can merge this :)

**Comment by aaron5670** [comment]
Any status updates? 😃

**Comment by scopsy** [comment]
@aaron5670 this looks great I will be merging this now and will release this in the upcoming version published in the next couple of days 🎉

**Review by scopsy — APPROVED** [review]

**Comment by scopsy** [comment]
@13colours thank you so much for the contribution! Merging this right now. <3

**Pull request merged**